cnf-certification-test/certification/suite.go

```diff
@@ -151,63 +148,67 @@ func testAllOperatorCertified(env *provider.TestEnvironment, validator certdb.Ce
 		channel := operatorsUnderTest[i].Channel
 		isCertified := validator.IsOperatorCertified(name, ocpMinorVersion, channel)
 		if !isCertified {
-			testFailed = true
-			logrus.Infof(
-				"Operator %s (channel %s) not certified for OpenShift %s.",
-				name,
-				channel,
-				ocpMinorVersion,
-			)
+			logrus.Infof("Operator %s (channel %s) not certified for OpenShift %s.", name, channel, ocpMinorVersion)
 			tnf.ClaimFilePrintf("Operator %s (channel %s) failed to be certified for OpenShift %s", name, channel, ocpMinorVersion)
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewOperatorReportObject(operatorsUnderTest[i].Namespace, operatorsUnderTest[i].Name, "Operator failed to be certified for OpenShift", false).
+				AddField(testhelper.OCPVersionType, ocpMinorVersion).
```

Comment:
If `OCPVersionType` and `OCPChannelType` are use as fields they should be defined under fields and not under types.

Reply:
Done.